playbooks/automation/check-patch.yml

```diff
@@ -4,3 +4,4 @@
 - import_playbook: "{{ playbook_dir }}/../cluster/{{ cluster | default('openshift') }}/config.yml"
 - import_playbook: "{{ playbook_dir }}/../cluster-login.yml"
 - import_playbook: "{{ playbook_dir }}/../kubevirt.yml"
+- import_playbook: "{{ playbook_dir }}/../storage.yml"
```

Comment:
I think that we should add a new playbook that deploys the different components, and import it in check-path.yml. It will allow users to deploy all the different components on an existing cluster.

Reply:
We're you thinking a components playbooks?  I think we can look at doing that in another PR.  I think there are multiple good approaches here.

Reply:
can't this be done today by simply running the desired playbooks in order?  I guess this is a general comment to make the repo easier for everyone to use and can be handled separately?